packages/postgresql-debversion-evr/postgresql-debversion-evr.spec

```diff
@@ -0,0 +1,63 @@
+%if 0%{?rhel} <= 7
+%global scl rh-postgresql12
```

Comment:
Should we only do so if no SCL is already set? In some cases the buildroot should already set it, right?

Reply:
Setting the `scl` via a global doesn't really work as you can see in the build:
```
error: line 5: Unknown tag: %scl_package postgresql-debversion-evr
```

You already need the SCL tools in the buildroot. There's a good chance Koji wasn't set up for that.